srtcore/buffer.cpp

```diff
@@ -192,14 +192,21 @@ void CSndBuffer::addBuffer(const char* data, int len, SRT_MSGCTRL& w_mctrl)
           log << "addBuffer: size=" << m_iCount << " reserved=" << m_iSize << " needs=" << size << " buffers for "
               << len << " bytes");
 
+    if (len <= 0)
+    {
+        LOGC(bslog.Error,
+              log << "IPE: addBuffer: %" << w_seqno << " #" << w_msgno << " non-positive message length " << len);
```

Comment:
Doesn't "negative" sound better? Or maybe simply "invalid" (as 0 is included)?

Reply:
I agree, "invalid" sounds better.